crates/e2e/src/xts.rs

```diff
@@ -325,10 +326,11 @@ where
             .build();
         self.client
             .tx()
-            .create_signed_offline(call, signer, params.into())
+            .create_partial_offline(call, params.into())
```

Comment:
Slight change in API introduced in https://github.com/paritytech/subxt/commit/b6b9ac65c76f2ad8246b4f55283d81be2c2ec000